Formula/newsboat.rb

```diff
@@ -12,6 +12,7 @@ class Newsboat < Formula
   end
 
   depends_on "asciidoc" => :build
+  depends_on "asciidoctor" => :build
   depends_on "docbook-xsl" => :build
```

Comment:
Asciidoctor doesn't need Docbook XSL, so this can be removed. [We removed dockbook-xsl from our CI](https://github.com/newsboat/newsboat/pull/747/files#diff-0f7067c31caedf7df2739c9b07efa122L55), too.